plugin/trino-memory/src/test/java/io/trino/plugin/memory/MemoryQueryRunner.java

```diff
@@ -93,7 +95,10 @@ public DistributedQueryRunner build()
                 queryRunner.createCatalog(CATALOG, "memory", memoryProperties.buildOrThrow());
 
                 queryRunner.installPlugin(new TpchPlugin());
-                queryRunner.createCatalog("tpch", "tpch", ImmutableMap.of());
+                queryRunner.createCatalog(
+                        "tpch",
+                        "tpch",
+                        ImmutableMap.of(TPCH_DOUBLE_TYPE_MAPPING_PROPERTY, DecimalTypeMapping.DOUBLE.name()));
```

Comment:
which BCT & BCST subclasses exercise BCT / BCST with tpch with decimals?

when will the existing tests migrated to use tpch with decimals?

Reply:
> which BCT & BCST subclasses exercise BCT / BCST with tpch with decimals?

I've updated TestTpchConnectorTest to use decimals now. I'm guessing that you're concerned about whether some BCT is run with tpch+decimals rather than which connectors are specifically doing what.

> when will the existing tests migrated to use tpch with decimals?

I think it is legitimate for other connectors and integration tests to configure tpch with doubles and continue to use that in their tests. There are tests which are specifically about double type and there are connectors which don't support decimals, so we definitely can't migrate everything. Whether we should migrate the tests which can be migrated depends on what's the benefit of making those changes.

Reply:
> I've updated TestTpchConnectorTest to use decimals now.

tpch doesn't support create table, so i guess this test class has many tests skipped.



> > when will the existing tests migrated to use tpch with decimals?
> 
> I think it is legitimate for other connectors and integration tests to configure tpch with doubles and continue to use that in their tests.

absolutely.

but the end state should be consistent: all connectors that can use decimals, should use decimals
and exceptions to the rule should be have rationale documented

if we cannot achieve consistency that way, maybe we should revert 0c589830cee42ea229b8f5c644b3070812fdbf1d instead